Detectors/TPC/workflow/CMakeLists.txt

```diff
@@ -63,6 +63,21 @@ o2_add_executable(calib-pedestal
                   SOURCES src/tpc-calib-pedestal.cxx
                   PUBLIC_LINK_LIBRARIES O2::TPCWorkflow)
 
+o2_add_executable(integrate-IDC
```

Comment:
I think IDC should be lower case for all of the following. It will be the executable names.